src/training_pipeline.py

```diff
@@ -105,7 +105,7 @@ def train(config: DictConfig) -> Optional[float]:
     )
     
     # model.model.bert_embedder.save_pretrained("/ssd1/jiahe/pretrained/scibert-synthetic-uncased-100k")
-    torch.save(model.model.state_dict(), "/ssd1/jiahe/pretrained/scibert-synthetic-uncased-full-100k")
+    torch.save(model.model.state_dict(), "checkpoints/scibert-synthetic-uncased-full-100k")
```

Comment:
Is this also an absolute path? Can we change to a relative path? Some user may store data/checkpoints/logs to different disk (e.g., /ssd1/XX/.parscit_cache/checkpoints/...).